build/xtask/src/dist.rs

```diff
@@ -462,7 +477,19 @@ pub fn package(
     )?;
 
     if let Some(signing) = toml.signing.get("combined") {
-        do_sign_file(signing, &out, &src_dir, "combined")?;
+        match ksymbol_table.get("HEADER") {
+            None =>  bail!("Didn't find header symbol -- does the image need a placeholder?"),
```

Comment:
(I think this error message is going to make someone very happy down the road, nicely done)